### PR TITLE
Add forti vuln module

### PR DESCRIPTION
**Pull request by Matie26**

### CODE REVIEW

File: autoreporter_addons/forti_vuln/reporter.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+from artemis.reporting.base.language import Language
+from artemis.reporting.base.report import Report
+from artemis.reporting.base.report_type import ReportType
+from artemis.reporting.base.reporter import Reporter
+from artemis.reporting.base.templating import ReportEmailTemplateFragment
+from artemis.reporting.utils import get_top_level_target
+
+
+class FortiVulnReporter(Reporter):  # type: ignore[misc]
```
Comment (kazet) [inline autoreporter_addons/forti_vuln/reporter.py:12]: not too broad ignore?

File: docker-compose.yml [hunk 2]
```diff
@@ -36,9 +36,22 @@ services:
     command: "python3 -m artemis.modules.karton_ssl_checks"
     profiles: [not-arm]
 
+  karton-forti_vuln:
```
Comment (kazet) [inline docker-compose.yml:39]: sort kartons alphaetically

File: forti_vuln/forti_vuln.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+    def vuln_check(self, host: str, port: int) -> int:
+        context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+        context.check_hostname = False
+        context.verify_mode = ssl.CERT_NONE
+
+        http_host = f"{host}:{port}"
+        self.log.info(f"forti vuln scanning {http_host}")
+
+        user_agent = ""
```
Comment (kazet) [inline forti_vuln/forti_vuln.py:66]: can we send an empty one? or sth generic, github link?

File: forti_vuln/forti_vuln.py [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+            if check == -1:
+                status = TaskStatus.ERROR
+                status_reason = "Could not send control request"
+            elif check == 1:
+                result.append("CVE-2024-21762")
+                status = TaskStatus.INTERESTING
+                status_reason = "Detected CVE-2024-21762 vulnerability"
+        else:
+            status = TaskStatus.ERROR
+            status_reason = "Omitted because service does not use ssl encryption"
```
Comment (kazet) [inline forti_vuln/forti_vuln.py:102]: why no check if no ssl?

File: autoreporter_addons/forti_vuln/reporter.py (hunk 1 above)
Comment (Matie26) [reply autoreporter_addons/forti_vuln/reporter.py:12]: fixed

File: docker-compose.yml (hunk 2 above)
Comment (Matie26) [reply docker-compose.yml]: now they are sorted

File: forti_vuln/forti_vuln.py (hunk 3 above)
Comment (Matie26) [reply forti_vuln/forti_vuln.py]: added default user agent just in case

File: forti_vuln/forti_vuln.py (hunk 4 above)
Comment (Matie26) [reply forti_vuln/forti_vuln.py]: added support for services without ssl

File: autoreporter_addons/forti_vuln/reporter.py (hunk 1 above)
Comment (Matie26) [reply autoreporter_addons/forti_vuln/reporter.py]: fixed

**Pull request merged**